api/types/src/convert.rs

```diff
@@ -805,10 +805,11 @@ impl<'a, R: ModuleResolver + ?Sized> MoveConverter<'a, R> {
             MoveTypeLayout::Signer => {
                 bail!("unexpected move type {:?} for value {:?}", layout, val)
             },
-            MoveTypeLayout::Tagged(tag, inner_layout) => match tag {
-                LayoutTag::IdentifierMapping(_) => {
-                    self.try_into_vm_value_from_layout(inner_layout, val)?
-                },
+
+            // Native types, e.g., aggregators, are currently used only at
+            // runtime, so we simply return an error here.
```

Comment:
I don't know why we don't allow signer, so kept the match case separately. I suspect this is because these are stored values in the DB, and you cannot really store signer (i think). Merged cases and added a comment about that.

Reply:
@zekun000 to confirm or redirect us :)